ui/desktop/electron-app/src/preload.js

```diff
@@ -26,6 +26,9 @@ contextBridge.exposeInMainWorld('terminal', {
   create: (vars) => {
     ipcRenderer.send('createTerminal', vars);
   },
+  remove: (callback, id) => {
+    ipcRenderer.removeListener(`removeTerminal-${id}`, callback);
```

Comment:
This would still need to be a `send` command and I don't think we'd have a callback

Reply:
Updated!